<ui version="4.0" >
 <class>ResultWidget</class>
 <widget class="QWidget" name="ResultWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTreeWidget" name="twResults" >
     <property name="indentation" >
      <number>0</number>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights" >
      <bool>true</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick" >
      <bool>false</bool>
     </property>
     <property name="columnCount" >
      <number>3</number>
     </property>
     <column>
      <property name="text" >
       <string>File</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Line</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Text</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QProgressBar" name="progressBar" >
       <property name="maximum" >
        <number>0</number>
       </property>
       <property name="value" >
        <number>15256</number>
       </property>
       <property name="textVisible" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnCancel" >
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblStatus" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Searching for files...</string>
       </property>
       <property name="buddy" >
        <cstring>btnCancel</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefine" >
       <property name="text" >
        <string>Refine Search...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove" >
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
